setup.py

```diff
@@ -1,6 +1,10 @@
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
 import os
-from setuptools import setup, find_packages
+
+from setuptools import find_packages, setup
+from six.moves import range
```

Comment:
@awais786 I missed this, but importing six here is a problem.  Because when edx-platform installs from scratch, six won't already exist in the virtual enviroment and the running of setup.py will fail.  In this case, it's okay to use range in both cases.  It's not that range moved but that it's behavior is different. But either behavior is fine in the way we use it here.

Can you make a PR to update and re-release django-wiki with this line removed?

Reply:
@feanil I will do this tomorrow.